terraform/deployments/govuk-publishing-platform/content_schemas.tf

```diff
@@ -1,5 +1,18 @@
+locals {
+  content_schemas = {
+    mount_point = {
+      sourceVolume  = "content_schemas",
+      containerPath = "/govuk-content-schemas",
```

Comment:
The `containerPath` is set by the environment variable `GOVUK_CONTENT_SCHEMAS_PATH` if I'm not mistaken. If this is indeed the case you might want to consider something like the following which I did in my spike: https://github.com/alphagov/govuk-infrastructure/pull/314/files#diff-59692f4e1dde43d64027a70e11c6c279842b47de85a8b1a86576619c69b2ffe6R40

Reply:
Thanks, I've changed the env vars to reference the local variable `local.content_schemas.mount_point.containerPath`.

Reply:
Would you also not need to do the same thing with this path?